### ISSUE TITLE
Break tests up into per-function files

### ISSUE THREAD

**Issue by skmgoldin** [ConsenSys/Tokens#102]
I like to organize tests like this: https://github.com/ConsenSys/PLCRVoting/tree/master/test

That way you can implement coverage trackers and more easily audit whether particular functions are being adequately covered.

**Comment by stvtortora** [comment]
Hey @skmgoldin. I see this issue is a bit old, but are you still interested in doing this?

**Comment by skmgoldin** [comment]
Not really, sorry! Busy with other things now. I also have no idea how used this tokens implementation still is.

What is the nature of your interest in this issue/project?
